CommandLine/libEGM/CMakeLists.txt

```diff
@@ -36,14 +46,23 @@ target_link_libraries(${LIB} PRIVATE pugixml)
 
 find_package(RapidJSON REQUIRED)
 include_directories(${LIB} PRIVATE ${RAPIDJSON_INCLUDE_DIRS})
-target_link_libraries(${LIB} PRIVATE rapidjson)
+if(MSVC)
+  target_link_libraries(${LIB} PRIVATE rapidjson)
```

Comment:
Why are we linking rapidjson at all? It's a header-only library, so there should be no need to link it anywhere.